qualang_tools/bakery/randomized_benchmark.py

```diff
@@ -2,7 +2,7 @@
 from qm.qua import *
```

Comment:
Undo all changed in this file.
This will be a test to make sure you didn't break backwards comptability.